src/modules/main/styles/ImportantContainer.style.tsx

```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles(
+  () => ({
+    container: {
+      minHeight: 455,
+      position: 'relative',
+    },
+    loading: {
+      position: 'absolute',
+      top: 'calc(50% - 30px)',
+      msTransform: 'translateY(-50%)',
+      transform: 'translateY(-50%),',
+    },
+  }),
+  { name: 'ImportantContainer' },
+);
```

Comment:
```suggestion
export const useStyles = makeStyles({
    container: {
      minHeight: 455,
      position: 'relative',
    },
    loading: {
      position: 'absolute',
      top: 'calc(50% - 30px)',
      msTransform: 'translateY(-50%)',
      transform: 'translateY(-50%),',
    },
  { name: 'ImportantContainer' },
);
```
you can just pass the configuration object in case you don't need theming

Reply:
fixed